File: benchmarks/python/fib.py
import sys, time
sys.setrecursionlimit(100000)
def fibonacci(n):
    if n > 1:
        return fibonacci(n - 1) + fibonacci(n - 2)
    return n
start = time.perf_counter()
result = fibonacci(35)
elapsed = time.perf_counter() - start
print(result)
print(elapsed)

File: benchmarks/js/fib.js
function fibonacci(n) {
  if (n > 1) return fibonacci(n - 1) + fibonacci(n - 2);
  return n;
}
const start = process.hrtime.bigint();
const result = fibonacci(35);
const elapsed = Number(process.hrtime.bigint() - start) / 1e9;
console.log(result);
console.log(elapsed);
